tests/utils.go

```diff
@@ -247,34 +247,34 @@ func failOnHttpClientCreation(err error) {
 	}
 }
 
-func GetRtDetails() auth.CommonDetails {
+func GetRtDetails() auth.ServiceDetails {
 	rtDetails := artifactoryAuth.NewArtifactoryDetails()
 	rtDetails.SetUrl(clientutils.AddTrailingSlashIfNeeded(*RtUrl))
 	setAuthenticationDetail(rtDetails)
 	return rtDetails
 }
 
-func GetDistDetails() auth.CommonDetails {
+func GetDistDetails() auth.ServiceDetails {
 	distDetails := distributionAuth.NewDistributionDetails()
 	distDetails.SetUrl(clientutils.AddTrailingSlashIfNeeded(*DistUrl))
 	setAuthenticationDetail(distDetails)
 	return distDetails
 }
 
-func setAuthenticationDetail(commonDetails auth.CommonDetails) {
-	if !fileutils.IsSshUrl(commonDetails.GetUrl()) {
+func setAuthenticationDetail(ServiceDetails auth.ServiceDetails) {
```

Comment:
I suggest changing the function argument to start with a lower case letter, to follow our conventions.

Reply:
Nice catch. Thanks!